cypress/integration/1230-doi_minting/1234-owner_drafts_a_doi_for_a_draft_publication/draft_doi_for_draft_registration.js

```diff
@@ -0,0 +1,50 @@
+import { Before } from 'cypress-cucumber-preprocessor/steps';
+import { Given, When, Then, And } from 'cypress-cucumber-preprocessor/steps';
+import { USER_DRAFT_DOI } from '../../../support/constants';
+
+const filename = 'example.txt';
+
+Before(() => {
+  cy.login(USER_DRAFT_DOI);
+  cy.startRegistrationWithFile(filename);
+  cy.get('[data-testid=registration-file-start-button]').should('be.enabled');
+  cy.get('[data-testid=registration-file-start-button]').click({ force: true });
+  cy.get('[data-testid=registration-title-field]').type('Draft registration requesting DOI');
+  cy.get('[data-testid=nav-tabpanel-resource-type]').click({ force: true });
+  cy.get('[data-testid=publication-context-type]').click({ force: true }).type(' ');
+  cy.get('[data-testid=publication-context-type-Journal]').click({ force: true });
+  cy.get('[data-testid=publication-instance-type]').click({ force: true }).type(' ');
+  cy.get('[data-testid=publication-instance-type-JournalArticle]').click({ force: true });
+  cy.get('[data-testid=button-save-registration]').click({ force: true });
+  cy.get('[data-testid=button-save-registration]').should('be.enabled');
+});
+
+Given('that an Owner views the Landing Page for their Registration', () => {
+  cy.get('[data-testid=my-registrations]').click({ force: true });
+});
+And('the Registration has status Draft', () => {
+  cy.get('[data-testid=unpublished-button]').click({ force: true });
+  cy.get('[data-testid^=registration-title]')
+    .filter(':contains("Draft registration requesting DOI")')

```

Comment:
hvis "Draft registration requesting DOI" skal brukes både her og på linje 12 kan det kanskje dras ut som en egen const (samme som er gjort for `filename`)